check_coordinates.py

```diff
@@ -0,0 +1,60 @@
+import json
+import pandas as pd
+import os
+import warnings
+import sys
+import csv
+
+# Inputs :
+# 1.csv_path = path of the CSV file with all the Biigle annotations 
+# 2.save_json_path = path of the directory where we want to save the
+#                    new CSV file with the annotations
+
+# Outputs :
+# The functions transform a CSV file with the Biigle annotations in 
+# a CSV file with the same annotations with coordinates in the 
+# image range
+    
+# the path to the CSV file
+path = sys.argv[1]
+# csv (output) path
+save_json_path = sys.argv[2]
+data = pd.read_csv(path)
+
+for row in data.itertuples():
+        # get width and heigth of the image
+        desired_dict = json.loads(row.attributes)
+        max_heigth =  desired_dict["height"]
+        max_width =  desired_dict["width"]
+        # get vector of coordinates
+        # row.points is a string, we want to transform it in an array of doubles
+        string_array = row.points
+        string_array = string_array.replace('[', '')
+        string_array = string_array.replace(']', '')
+        splitted_string = string_array.split (",")
+        # desired_array is an array of doubles, like [x1, y1, x2, y2, ecc]
+        desired_array = [float(numeric_string) for numeric_string in splitted_string]
```

Comment:
This should include a check for cirlces (lists with exactly three elements). The third element of the list (radius of the circle) should not be modified by the script.

Reply:
OK, I had not included the circles as I only work with polygons. I added an if condition for circles and an else for all other shapes. I hope this is sufficient